lib/os/thread_posix.cpp

```diff
@@ -56,43 +70,44 @@ namespace pstore {
             // - two argument form supported by Linux.
             // - three argument form supported by NetBSD.
             // - the slightly differently named form used in FreeBSD.
-#    ifdef __FreeBSD__
+            int err = 0;
+#    if !defined(PTHREAD_SETNAME_NP) || !defined(PTHREAD_GETNAME_NP)
```

Comment:
As above. It would probably be a good idea to add another macro for this expression.

Reply:
Fixed in the commit ed5cf97.